web/yaamp/modules/site/common_results.php

```diff
@@ -190,18 +190,32 @@ function cmp($a, $b)
 	if ($algo_selected == $algo)	{
 		echo "<tr>";
 		echo "<td colspan='13'>";
-		echo "<table>";
-		echo "<thead>";
-		echo "<tr>";
-		echo "<th data-sorter='numeric' align='left">PID</th>";
-		echo "<th data-sorter='numeric' align='left">Time</th>";
-		echo "<th data-sorter='numeric' align='left">Started</th>";
-		echo "<th data-sorter='numeric' align='left">Workers</th>";
-		echo "<th data-sorter='numeric' align='left">Port</th>";
-		echo "<th data-sorter='numeric' align='left">Symbol</th>";
-		echo "<th data-sorter='numeric' align='left">url</th>";
-		echo "<th data-sorter='numeric' align='left">fds</th>";
-		echo "</tr>";
+		showTableSorter('stratumstable', '{
+			tableClass: "dataGrid",
+			widgets: ["Storage","saveSort"],
+			textExtraction: {
+				1: function(node, table, cellIndex) { return $(node).attr("data"); },
+				5: function(node, table, cellIndex) { return $(node).attr("data"); }
+			},
+			widgetOptions: {
+				saveSort: true
+			}}');
+		echo <<<END
+		<table>
+		<thead>
+		<tr>
+		<th data-sorter='numeric' align='left">PID</th>
+		<th data-sorter='numeric' align='left">Time</th>
+		<th data-sorter='numeric' align='left">Started</th>
+		<th data-sorter='numeric' align='left">Workers</th>
+		<th data-sorter='numeric' align='left">Port</th>
+		<th data-sorter='numeric' align='left">Symbol</th>
+		<th data-sorter='numeric' align='left">url</th>
+		<th data-sorter='numeric' align='left">fds</th>
+		</tr>
+		</thead>
+		<tbody>
+		END;
 		$stratums_details_list = dbolist("SELECT * FROM stratums WHERE algo:=algo='$algo_selected'");
 		foreach ($stratums_details_list as $stratums_details)	{
 			echo "<tr>";
```

Comment:
Yep, you need to fill data="$time" in the time td... and use the datetoa2() function for the content